### PR TITLE
add Keras example

### PR DESCRIPTION
**Pull request by Asakuri**

### CODE REVIEW

File: example/keras/README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    "jobName": "keras_tensorflow_backend_mnist",
+    "image": "paiexample/pai.example.keras.tensorflow",
+    "taskRoles": [
+        {
+            "name": "mnist",
+            "taskNumber": 1,
+            "cpuNumber": 4,
+            "memoryMB": 8192,
+            "gpuNumber": 1,
+            "command": "python mnist_cnn.py"
```
[the remark this answers is not in the document]
Comment (Asakuri) [reply example/keras/README.md]: set in dockerfile

File: example/keras/keras.cntk_backend.mnist.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "jobName": "keras_tensorflow_backend_mnist",
+    "image": "paiexample/pai.example.keras.cntk",
+    "taskRoles": [
+        {
+            "name": "mnist",
+            "taskNumber": 1,
+            "cpuNumber": 4,
+            "memoryMB": 8192,
+            "gpuNumber": 1,
+            "command": "python mnist_cnn.py"
```
[the remark this answers is not in the document]
Comment (Asakuri) [reply example/keras/keras.cntk_backend.mnist.json]: set in dockerfile

File: example/keras/Dockerfile.example.keras.cntk_backend [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+# install Keras using pip
+RUN pip install keras cntk-gpu
+
+# clone Keras examples
+RUN git clone git clone https://github.com/keras-team/keras.git 
+
+ENV KERAS_BACKEND cntk
+
+WORKDIR /keras/examples
```
Comment (abuccts) [inline example/keras/Dockerfile.example.keras.cntk_backend]: The `WORKDIR` in base image is `/root`, so keras is cloned into `/root/keras`?

File: example/keras/Dockerfile.example.keras.cntk_backend [hunk 4]
```diff
@@ -0,0 +1,14 @@
+FROM aiplatform/pai.run.cntk
+
+# install git
+RUN apt-get -y update && apt-get -y install git
+
+# install Keras using pip
+RUN pip install keras cntk-gpu
+
+# clone Keras examples
+RUN git clone git clone https://github.com/keras-team/keras.git
```
Comment (abuccts) [inline example/keras/Dockerfile.example.keras.cntk_backend]: double `git clone`

File: example/keras/Dockerfile.example.keras.tensorflow_backend [hunk 5]
```diff
@@ -0,0 +1,14 @@
+FROM aiplatform/pai.run.tensorflow
+
+# install git
+RUN apt-get -y update && apt-get -y install git
+
+# install Keras using pip
+RUN pip install keras
+
+# clone Keras examples
+RUN git clone git clone https://github.com/keras-team/keras.git
```
Comment (abuccts) [inline example/keras/Dockerfile.example.keras.tensorflow_backend]: double `git clone`

File: example/keras/Dockerfile.example.keras.cntk_backend (hunk 3 above)
Comment (Asakuri) [reply example/keras/Dockerfile.example.keras.cntk_backend]: updated, but workdir in latest pulled aiplatform/pai.run.tensorflow image is `/`

**Pull request merged**